db/fixtures/ae_datastore/ManageIQ/Deployment/ContainerProvider/System/StateMachine/Deployment.class/__methods__/add_provider.rb

```diff
@@ -1,21 +1,17 @@
+gem "net-ssh", "=4.0.0.alpha4"
```

Comment:
I'm not really knowledgable on our usage of net/ssh so I'll let @gmcculloug review those changes.  🙇